src/jit/emitxarch.cpp

```diff
@@ -10099,6 +10127,8 @@ BYTE* emitter::emitOutputAM(BYTE* dst, instrDesc* id, code_t code, CnsVal* addc)
             switch (id->idInsFmt())
             {
                 case IF_RWR_ARD:
+                case IF_RRW_ARD:
+                case IF_RWR_RRD_ARD:
```

Comment:
I wasn't actually getting GCStress failures for the `RWR_RRD_ARD` (as well as the corresponding `SRD`, `MRD`, and `RRD`) entries... However, we are definitely executing SIMD instructions that write integer registers and that use those formats....

Anyone know if there is something I'm missing here?

Reply:
You may just be getting (un)lucky, as the written registers won't always contain GC refs.